test/library/randomized_benchmarking/test_interleaved_rb.py

```diff
@@ -268,14 +269,14 @@ def test_interleaved_circuit_is_decomposed(self):
 
     def test_interleaving_cnot_gate_with_non_supported_direction(self):
         """Test if fails to interleave cx(1, 2) for backend that support only cx(2, 1)."""
-        del self.backend.target["cx"][(1, 2)]  # make support only cx(2, 1)
+        backend = GenericBackendV2(3, coupling_map=[[0, 1], [2, 1]])
```

Comment:
This new way is definitely a much safer way of doing things - `Target` was open for additions through its methods, but mutating the data structures directly would be _really_ fragile; there's a lot of cross-dependencies between various internal structures, not all of which were public.

`Target` was never supposed to allow deletions, so a rebuild is much safer.